vega_sim/api/data.py

```diff
@@ -282,8 +282,9 @@ class MarketData:
     market_value_proxy: float
     price_monitoring_bounds: list
     liquidity_provider_fee_share: list
+    liquidity_sla: list
```

Comment:
Would give some useful type hinting when working with the `MarketData` dataclass.
```suggestion
    liquidity_sla: List[LiquidityProviderSLA]
```